django/http/request.py

```diff
@@ -461,6 +461,27 @@ def parse_header_name(cls, header):
             return None
         return header.replace("_", "-").title()
 
+    @classmethod
+    def to_wsgi_names(cls, headers):
+        return {
+            HttpHeaders.to_wsgi_name(header_name): value
+            for header_name, value in headers.items()
+        }
+
+    @classmethod
+    def to_wsgi_name(cls, header):
```

Comment:
Put this above `to_wsgi_names()` to keep the methods that act on multiple headers together.